specification/window.browser.bs

```diff
@@ -52,6 +52,36 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     it MUST be used only for WebExtension related functionality.
   </section>
 
+  <section>
+    For backwards compatibility with older versions of WebExtensions that used
+    <code>chrome</code> as a global identifier for WebExtension related methods,
+    a UA MAY alias individual properties of {{browser}} to <code>chrome</code>.
+    {{browser}} and <code>chrome</code> SHOULD NOT be direct aliases. Instead,
+    individual subproperties SHOULD be alaised. Any modification of a property
+    of {{browser}} SHOULD be reflected on the same subproperty of <code>chrome</code>,
+    and vice versa.
```

Comment:
I thought that we were going to specify that they may be direct aliases. Here is a different phrasing that captures this, and also defines the relation more precisely:

```suggestion
    {{browser}} and <code>chrome</code> MAY be direct aliases of each other.
    When {{browser}} and <code>chrome</code> are distinct objects, each initial
    property of {{browser}} SHOULD be defined on the <code>chrome</code> object,
    and each property descriptor on the <code>chrome</code> object SHOULD return
    the same value as the corresponding property on the {{browser}} object.
```

The original wording said "Any modification of a property of {{browser}} SHOULD be reflected on the same subproperty of <code>chrome</code> and vice versa." Besides the direct value alias, this would (unintentionally?) also imply:

- `browser.tabs = 1; assert(chrome.tabs === 1);` (the tabs property has been modified)
- `browser.whatever = 2; assert(chrome.whatever === 2);` ("Any modification of a property" does not say that user-defined properties aren't mirrored)
- `chrome.csi = 2; assert(browser.csi === 2);`  ("Any modification of a property" + "and vice versa" implies that `chrome` and `browser` should share the same set of properties?)